builtin/logical/pkiext/pkiext_binary/acme_test.go

```diff
@@ -60,13 +59,14 @@ func SubtestACMECertbot(t *testing.T, cluster *VaultPkiCluster) {
 		LogStdout:     logStdout,
 		LogStderr:     logStderr,
 	})
+	require.NoError(t, err, "failed creating service runner")
 
 	ctx := context.Background()
 	result, err := runner.Start(ctx, true, false)
 	require.NoError(t, err, "could not start container")
 	require.NotNil(t, result, "could not start container")
 
-	defer runner.Stop(context.Background(), result.Container.ID)
+	defer ignoreError(runner.Stop(context.Background(), result.Container.ID))
```

Comment:
Does `defer` check errors? I was under the impression it didn't.

Reply:
No I was just silencing warnings from Goland.

Reply:
and yeah, a last minute fix on Friday. This turned out to be the CI problem. Reverted.